src/imitation/data/buffer.py

```diff
@@ -65,41 +65,56 @@ def __init__(
         self._idx = 0
 
     @classmethod
-    def from_data(cls, data: Dict[str, np.ndarray]) -> "Buffer":
-        """Constructs and return a Buffer containing only the provided data.
+    def from_data(
+        cls,
+        data: Dict[str, np.ndarray],
+        capacity: Optional[int] = None,
+        truncate_ok: bool = False,
+    ) -> "Buffer":
+        """Constructs and return a Buffer containing the provided data.
 
-        The returned Buffer is at full capacity and ready for sampling.
+        Shapes and dtypes are automatically inferred.
```

Comment:
Could you add one or two examples showing usages of `capacity` and `truncate_ok`? At first I was really confused why you need both. I think the use cases are:
  - Specify `capacity` and `truncate_ok=True` if you are specifying a smaller capacity than your data.
  - Specify `capacity` and `truncate_ok=False` if you are specifying a larger capacity than you intend your data to fill, and may call `store` subsequently to add more data.
  - Do not specify `capacity` if you want the buffer to be the same size as your data. In this case `truncate_ok` has no effect.

Reply:
Added examples, looks like this.
![image](https://user-images.githubusercontent.com/1750835/86081769-fa6cc500-ba4a-11ea-9869-d5c6f74ff267.png)